File: Java/Basic Syntax/src/SymetricNumbers.java
import java.util.Scanner;

public class SymetricNumbers {
    public static void main(String[] args) {
        Scanner scanner=new Scanner(System.in);

        int n= Integer.parseInt(scanner.nextLine());

        for (int i = 1; i <=n ; i++) {
            String numberAsString=i+"";
            if(isSymetric(numberAsString)){
                System.out.println(i+" ");
            }
        }
    }

    private static boolean isSymetric(String number){
        //String reversedNumber="";
        StringBuilder sb=new StringBuilder();
        for(int i=number.length()-1;i>=0;i--){
            sb.append(number.charAt(i));
        }

        if(number.equals(sb.toString())){
            return true;
        }
        return false;
    }
}

File: JavaScript/Basic web/JS Syntax Lab/SymetricNumbers.js
function symetricNumbers(arr) {
    let endNumber=Number(arr[0]);
    let result="";
    for (let i = 1; i <=endNumber ; i++) {
        if (isSymetric(i.toString())){
            result+=i+" ";
        }
    }
    console.log(result);

    function isSymetric(number) {
        for (let j = 0; j <number.length/2 ; j++) {
            if(number[j]!=number[number.length-j-1]){
                return false;
            }
        }
        return true;
    }
}
symetricNumbers(["1000"])